acra-censor/acra-censor_configuration_provider.go

```diff
@@ -112,7 +107,7 @@ func (acraCensor *AcraCensor) LoadConfiguration(configuration []byte) error {
 			go queryCaptureHandler.Start()
 			acraCensor.AddHandler(queryCaptureHandler)
 		default:
-			break
+			acraCensor.logger.Errorln("Unexpected handler in configuration")
```

Comment:
would be good to return some error, not just log. with error, we disallow to run server with misconfigured censor